index.php

```diff
@@ -91,9 +133,11 @@
     <nav class="nav">
         <ul class="nav__list container">
             <!--заполните этот список из массива категорий-->
+            <?php foreach($lots as $key => $val): ?>
```

Comment:
Тут нужно итерировать список категорий из переменной `$categories`. Также, стоит убирать `$key =>` где оно не требуется и давать осмысленные имена переменным вместо `val`.
```suggestion
            <?php foreach($categories as $category): ?>
```